### ISSUE TITLE
Upgrading to use Kotlin 1.3 breaks Gradle sync

### ISSUE THREAD

**Issue by NinjaRat** [boxheed/gradle-pater-build-plugin#11]
Hi,

Using the Pater Plugin works when using Kotlin 1.2.71, however it breaks and cannot sync if updating to the new version 1.3.11.

* What went wrong:
A problem occurred configuring project ':app'.
> Cannot change attributes of configuration ':app:debugCompile' after it has been resolved

However I don't know exactly where it breaks.

Best regards

**Comment by boxheed** [comment]
@NinjaRat I'll take a look, could you provide me with a build file that demonstrates this please. Also I'm not familiar with Kotlin any hints and tips I need to consider?

**Comment by NinjaRat84** [comment]
Hi!

Thanks for a quick response, this is my current config. Hope it helps :D

build.gradle in Root
```
buildscript {
    ext.kotlin_version = '1.2.71'
    
    dependencies {
        classpath 'com.android.tools.build:gradle:3.2.1'
        classpath 'io.github.robwin:jgitflow-gradle-plugin:0.6.0'
        classpath 'com.libraries.android:CommonApp:1.0.0'
        
        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
        classpath "org.jetbrains.kotlin:kotlin-android-extensions:$kotlin_version"

        // NOTE: Do not place your application dependencies here; they belong
        // in the individual module build.gradle files 
    }
}
```

Then in app:build.gradle

```
apply plugin: 'com.android.application'

apply plugin: 'com.fizzpod.pater-build'
apply plugin: 'io.github.robwin.jgitflow'

apply plugin: 'kotlin-android'
apply plugin: 'kotlin-android-extensions'
apply plugin: 'kotlin-kapt'

dependencies {
    implementation fileTree(dir: 'libs', include: ['*.jar'])

    // Kotlin
    implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
}
```

In our Common Root:build.gradle the following is defined:
`dependencies {
        classpath 'com.fizzpod:gradle-pater-build-plugin:2.0.0'
        classpath 'io.github.robwin:jgitflow-gradle-plugin:0.6.0'
    }`

and then in the actual CommonApp build.gradle has, this is the module containing META-INF directory
```
apply plugin: 'java'
apply plugin: 'maven-publish'

dependencies {
    compile 'com.fizzpod:gradle-pater-build-plugin:2.0.0'
}
```

And somehow it seems to work when using Kotlin 1.2.71, but breaks if upgraded to 1.3.11.
**Note**: If I comment out the `apply plugin: 'com.fizzpod.pater-build'` in app:builg.gradle for the app, it sync correct with Kotlin 1.3, only breaks when using with Common META-INF files.

Best regards

**Comment by boxheed** [comment]
@NinjaRat 

I've had a look and cannot get these files to work, I really need a simplified project that demonstrates the error as well as the steps to reproduce (bear in mind I'm not familiar with Kotlin nor android development or Android studio).

Is this an error happening in Android Studio? Does the error happen when running a build on the commandline? 

Can you also provide the Java version, and the gradle version as well.

**Comment by NinjaRat84** [comment]
@boxheed 

I've continued looking into the issue and found that the issue was not concerning the gradle-pater-plugin like I thought. Since we had methods for sharing to share build to Fabric Beta (Android), this seemed to break the build.

Since we are moving to using Fastane instead, these files was removed and it synced successfully again.

Sorry for any inconvenience, and thanks for all the support.

Best regards
